docker/developers/.env

```diff
@@ -0,0 +1,7 @@
+# Older versions of the browsers require older versions of the selenium hub
+SELENIUM_HUB_VERSION=4.8.2
+
+# This sets common variables used throughout the compose stack
+SELENIUM_FIREFOX_VERSION=112.0
+SELENIUM_CHROME_VERSION=105.0
+SELENIUM_EDGE_VERSION=105.0
```

Comment:
This is good to start with, but I wonder if we can read the versions from [browsers.json ](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/ui/browsers.json)to have a single source of truth for browser versions.

Reply:
The docker compose file cannot execute before its own definition. This file is static and should be true for all developers. You should update this file along with the browsers.json. If you want to be pedantic, write a lint test to ensure they match.